js-miniapp-sdk/README.md

```diff
@@ -411,6 +411,60 @@ miniApp.chatService.sendMessageToMultipleContacts(messageToContact)
 
 ## Advanced Usage
 
+### Errors management
+
+#### Access Token error
+
+Error messages sent to the bridge when a `getAccessToken` call is failing should always be in the following JSON format:
+
+````json
+{
+  "message": "error_message",
+  "type": "error_type_key"
+}
+````
+At the moment, the SDK will throw a `MiniAppError` (and subclasses), subclassing `Error` type, only on `getAccessToken` failures, based on `type` value.
+If the error `type` value is not supported, `Error.name` field will automatically be set to `Other`.
+
+Here are the `type` values currently supported and their default message:
+
+| Error Type | Message |
+| ---- | ---- |
+| `AudienceNotSupportedError` | `The value passed for 'audience' is not supported.` |
+| `ScopesNotSupportedError` | `The value passed for 'scopes' is not supported.` |
+| `AuthorizationFailureError` | _a message should be provided_ |
+| `Other` |  _a message should be provided_ |
```

Comment:
This also seems like implementation details. Users will receive an instance of `AuthorizationFailureError`, so they shouldn't need to know the `type`, right?

Reply:
You're right, the example is self-sufficient